clumio/test/acc_test_lambda.py

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright 2021. Clumio, Inc.
+#
+import logging
+import json
+import os
+
+import boto3
+from typing import Any
+
+"""Lambda event handler for acceptance test of clumio_callback_resource."""
+
+
+def clumio_event_handler(event: dict[str, Any], _: Any) -> None:
+    """Event handler to process the clumio event."""
+    logger = logging.Logger(name="process_event_logger")
+    resource_props = json.loads(
+        event.get('Records', [])[0].get('Sns', {}).get('Message', '{}')).get(
+        'ResourceProperties')
+    if type(resource_props) is not dict:
+        raise Exception('ResourceProperties must be of type dictionary.')
+
+    bucket_name_key = 'BUCKET_NAME'
+    bucket_name = os.getenv(bucket_name_key)
+    raise_exception_if_empty(bucket_name_key, bucket_name)
+    account_id_key = 'AccountId'
+    account_id = resource_props.get(account_id_key)
+    raise_exception_if_empty(account_id_key, account_id)
+    token_key = 'Token'
+    token = resource_props.get(token_key)
+    raise_exception_if_empty(token_key, token)
+    region_key = 'Region'
+    region = resource_props.get(region_key)
+    raise_exception_if_empty(region_key, region)
+    event_publish_time_key = 'EventPublishTime'
+    event_publish_time = resource_props.get(event_publish_time_key)
+    raise_exception_if_empty(event_publish_time_key, event_publish_time)
+    canonical_user_key = 'CanonicalUser'
+    canonical_user = resource_props.get(canonical_user_key)
+    raise_exception_if_empty(canonical_user_key, canonical_user)
+    aws_region_key = 'AWS_REGION'
+    region = os.getenv(aws_region_key)
+    raise_exception_if_empty(aws_region_key, region)
+    client = boto3.client('s3')
+    success_msg = {
+        'Status': "SUCCESS"
+    }
+    object_key = (
+        f'acmtfstatus/{account_id}/{region}/{token}/clumio-status-{event_publish_time}.json')
+    try:
+        client.put_object(
+            Bucket=bucket_name, Key=object_key, Body=bytes(json.dumps(success_msg), 'utf-8'))
+    except Exception as ex:
```

Comment:
This technically doesn't break the style guide because it re-raises the original exception.  However, catching the specific errors (such as `ClientError`) and handling them as appropriate would be more consistent with the rest of the Python codebase.

* https://boto3.amazonaws.com/v1/documentation/api/latest/guide/error-handling.html

Reply:
fixed.